website/src/components/wordpress.tsx

```diff
@@ -53,13 +64,22 @@ const parseOptions: HTMLReactParserOptions = {
     if ("data" in node) {
       const segments = node.data.match(style)?.filter((x) => x !== undefined)
       if (segments) {
-        console.log(segments)
-
         if (segments?.length === 4) {
```

Comment:
The only difference in these two conditional branches is whether segments[3] is null or not. So I would combine them under a condition like `segments.length >= 3` and check if the last element is null or not when parsing it, something like `segments.length >= 4 ? parseInt(segments[3]) : null`.